similar/main/game.cpp

```diff
@@ -252,6 +252,10 @@ void init_cockpit()
 	}
 #endif
 
+	// Hack to keep stereo formats remaining fullscreen
```

Comment:
Rather than do this hack, I suggest trapping the change properly.  You can use a write-access breakpoint in the debugger to run until the value of interest is changed, then trap in.  This lets the game/debugger find the site for you, rather than searching via code inspection.

If using gdb, look at `watch` to monitor variables.  You probably also want to run the game with keyboard grabs disabled (command line option `-no-grab` can do this on a temporary basis).  Otherwise, you may find the game paused by the debugger and all your keyboard input sent to the game, which cannot respond because it is paused.

Reply:
Thanks for the debugger tips. Tried a gdb session earlier while in fullscreen mode, but could not regain control after hitting a breakpoint. Perhaps gdb session by remote terminal might be more amenable to fullscreen modes?

Reply:
Was able to do gdb session from remote terminal to trap the change in cockpit view. Player death and level end sequences will call select_cockpit(CM_LETTERBOX), so relocated stereo conditional to skip cockpit views inside that routine.